.github/workflows/build_docker.yml

```diff
@@ -57,7 +55,7 @@ jobs:
 
       - name: Login to Amazon ECR
         id: login-ecr-public
-        uses: aws-actions/amazon-ecr-login@v1
+        uses: karlonovak/amazon-ecr-login@v2.0.0
```

Comment:
I would prefer that we stay on the AWS-maintained version of this action - could you elaborate on the reason for changing this? 

I believe the latest versions of the original action shouldn't trigger any of the deprecation warnings, the requisite changes were made to remove `set-output` (see [here](https://github.com/aws-actions/amazon-ecr-login/issues/358#issuecomment-1295750476)) and upgrade to Node 16 (see [here](https://github.com/aws-actions/amazon-ecr-login/commit/29516697cce506511bbd2ab14eec27dcfb5626b9)).

Reply:
not a problem I reverted it back to stay within scope.